Gemfile

```diff
@@ -28,6 +28,16 @@ group :test do
   gem 'puppet-strings', '0.4.0',                                    :require => false
   gem 'rubocop-rspec', '~> 1.6',                                    :require => false if RUBY_VERSION >= '2.3.0'
   gem 'json_pure', '<= 2.0.1',                                      :require => false if RUBY_VERSION < '2.0.0'
+  gem 'rake' , '11.2.2'
+  gem 'mime-types' , '2.99.2'
+  gem 'unicode-display_width','1.1.0'
+  gem 'parser', '2.3.1.2'
+  gem 'mocha', '1.1.0'
```

Comment:
I was able to successfully execute `bundle exec rake test` by just pinning `mocha` to version `1.1.0`.